stable/redis/templates/redis-slave-deployment.yaml

```diff
@@ -71,6 +78,16 @@ spec:
         image: {{ template "redis.image" . }}
         imagePullPolicy: {{ .Values.image.pullPolicy | default "" | quote }}
 {{- $command := default .Values.master.command .Values.slave.command }}
+        {{- /* Include master securityContext.runAsUser if slave securityContext.runAsUser not defined */ -}}
+        {{- if (.Values.slave.securityContext | default .Values.master.securityContext) }}
+        securityContext:
+          runAsNonRoot: true
```

Comment:
It is necessary to add the `runAsNonRoot` property if we are already specifying `runAsUser`?
In any case, I wouldn't add it as in some cases it might be necessary to run the containers as the root user for what you would use `runAsUser: 0`.

Reply:
Good point. Thanks for suggesting that.